### ISSUE TITLE
Errors and warnings output when parsing file including <<resourcePathName>>

### ISSUE THREAD

**Issue by glodieu** [raml-org/raml-js-parser-2#639]
Using the following RAML file:
```
#%RAML 1.0
title: Example API
version: v1
resourceTypes:
  collection:
    usage: 
        This resourceType should be used for any 
        collection of items
    description: 
        The collection of <<resourcePathName>>
    get:
      description: Get all <<resourcePathName>>, optionally filtered
    post:
      description: Create a new <<resourcePathName | !singularize>>


traits:
  secured:
    usage: 
       Apply this to any method that needs to be 
       secured
    description: Some requests require authentication.
    headers:
      access_token:
        description: Access Token
        example: 5757gh76
        required: true
        type: string

/users:
  type: collection
  is: [ secured ]
  get:
  post:
```

```
Parser outputs there messages:
[xmldom error]	element parse error: Error: invalid tagName:<resourcePathName 
@#[line:1,col:19]
[xmldom warning]	unclosed xml attribute 
@#[line:1,col:20]
[xmldom error]	element parse error: Error: invalid tagName:<resourcePathName 
@#[line:1,col:9]
[xmldom warning]	unclosed xml attribute 
@#[line:1,col:10]
[xmldom error]	element parse error: Error: invalid tagName:<resourcePathName 
@#[line:1,col:14]
[xmldom warning]	attribute "|" missed value!! "|" instead2!! 
@#[line:1,col:15]
[xmldom error]	element parse error: Error: invalid attribute:| 
@#[line:1,col:15]
```

**Comment by glodieu** [comment]
I'm using "raml-1-parser": "1.1.14"

**Comment by glodieu** [comment]
Another error messages:
```
#%RAML 0.8
title: Products API
version: v0.1
baseUri: 'http://mocksvc.mulesoft.com/mocks/a86239e0-d3af-48c6-9fd5-f8edc7c35d69'
mediaType: application/json
protocols:
  - HTTP
  - HTTPS
documentation:
  - title: Products API
    content: ''
/custom:
  displayName: custom
  post:
    body:
      application/x-www-form-urlencoded:
        formParameters:
          AWSAccessKeyId:
            type: string
            description: The AWS Access Key ID of the owner of the bucket who grants an Anonymous user access for a request that satisfies the set of constraints in the Policy.
          acl:
            type: string
            description: 'Specifies an Amazon S3 access control list. If an invalid access control list is specified, an error is generated.'
/products:
  displayName: products
  post:
    description: Create a product
    body:
      application/json:
        example: |-
          {
            "product": {
              "id": "1",
              "name": "Product One",
              "description": "This is the full description of the product.",
              "url": "http://example.com",
              "image": "http://example.com/image.jpg",
              "thumbnailUrl": "http://example.com/image-thumb.jpg",
              "keywords": "western, cowboy",
              "brand": "Brand Name",
              "color": "Black",
              "itemCondition": "New",
              "manufacturer": "Manufacturer Name",
              "model": "Black",
              "sku": "SKU #",
              "weight": "12 pounds",
              "width": "12 inches",
              "height": "12 inches",
              "depth": "12 inches"
            }
          }
        schema: |-
          {
            "type": "object",
            "properties": {
              "product": {
                "type": "object",
                "required": [
                  "name",
                  "description",
                  "url",
                  "image"
                ],
                "properties": {
                  "name": {
                    "type": "string"
                  },
                  "description": {
                    "type": "string"
                  },
                  "url": {
                    "type": "string"
                  },
                  "image": {
                    "type": "string"
                  },
                  "thumbnailUrl": {
                    "type": "string"
                  },
                  "keywords": {
                    "type": "string"
                  },
                  "brand": {
                    "type": "string"
                  },
                  "color": {
                    "type": "string"
                  },
                  "itemCondition": {
                    "type": "string"
                  },
                  "manufacturer": {
                    "type": "string"
                  },
                  "model": {
                    "type": "string"
                  },
                  "sku": {
                    "type": "string"
                  },
                  "weight": {
                    "type": "string"
                  },
                  "width": {
                    "type": "string"
                  },
                  "height": {
                    "type": "string"
                  },
                  "depth": {
                    "type": "string"
                  }
                }
              }
            },
            "required": [
              "product"
            ]
          }
    headers:
      X-Tracking-Example:
        type: string
        description: You can specify request headers like this
        enum:
          - accounting
          - payroll
          - finance
        required: false
        example: accounting
        displayName: X-Tracking-Example
  get:
    description: Get a list of products
    responses:
      '200':
        body:
          application/json:
    queryParameters:
      q:
        type: string
        description: Search phrase to look for products
        required: false
        displayName: q
  '/{productId}':
    displayName: '{productId}'
    uriParameters:
      productId:
        displayName: productId
        type: string
    get:
      description: Get a single product
      responses:
        '200':
          body:
            application/json:
              schema: '{}'
              example: ''
      queryParameters:
        productId:
          type: integer
          description: The ID of the product
          required: true
          displayName: productId
    put:
      description: Updates a single product
      body:
        application/json:
          example: |-
            {
              "product": {
                "id": "1",
                "name": "Product One",
                "description": "This is the full description of the product.",
                "url": "http://example.com",
                "image": "http://example.com/image.jpg",
                "thumbnailUrl": "http://example.com/image-thumb.jpg",
                "keywords": "western, cowboy",
                "brand": "Brand Name",
                "color": "Black",
                "itemCondition": "New",
                "manufacturer": "Manufacturer Name",
                "model": "Black",
                "sku": "SKU #",
                "weight": "12 pounds",
                "width": "12 inches",
                "height": "12 inches",
                "depth": "12 inches"
              }
            }
          schema: |-
            {
              "type": "object",
              "properties": {
                "product": {
                  "type": "object",
                  "required": [
                    "name",
                    "description",
                    "url",
                    "image"
                  ],
                  "properties": {
                    "name": {
                      "type": "string"
                    },
                    "description": {
                      "type": "string"
                    },
                    "url": {
                      "type": "string"
                    },
                    "image": {
                      "type": "string"
                    },
                    "thumbnailUrl": {
                      "type": "string"
                    },
                    "keywords": {
                      "type": "string"
                    },
                    "brand": {
                      "type": "string"
                    },
                    "color": {
                      "type": "string"
                    },
                    "itemCondition": {
                      "type": "string"
                    },
                    "manufacturer": {
                      "type": "string"
                    },
                    "model": {
                      "type": "string"
                    },
                    "sku": {
                      "type": "string"
                    },
                    "weight": {
                      "type": "string"
                    },
                    "width": {
                      "type": "string"
                    },
                    "height": {
                      "type": "string"
                    },
                    "depth": {
                      "type": "string"
                    }
                  }
                }
              },
              "required": [
                "product"
              ]
            }
      responses:
        '200':
          body:
            application/json:
              schema: |-
                {
                  "type": "object",
                  "properties": {
                    "product": {
                      "type": "object",
                      "required": [
                        "name",
                        "description",
                        "url",
                        "image"
                      ],
                      "properties": {
                        "name": {
                          "type": "string"
                        },
                        "description": {
                          "type": "string"
                        },
                        "url": {
                          "type": "string"
                        },
                        "image": {
                          "type": "string"
                        },
                        "thumbnailUrl": {
                          "type": "string"
                        },
                        "keywords": {
                          "type": "string"
                        },
                        "brand": {
                          "type": "string"
                        },
                        "color": {
                          "type": "string"
                        },
                        "itemCondition": {
                          "type": "string"
                        },
                        "manufacturer": {
                          "type": "string"
                        },
                        "model": {
                          "type": "string"
                        },
                        "sku": {
                          "type": "string"
                        },
                        "weight": {
                          "type": "string"
                        },
                        "width": {
                          "type": "string"
                        },
                        "height": {
                          "type": "string"
                        },
                        "depth": {
                          "type": "string"
                        }
                      }
                    }
                  },
                  "required": [
                    "product"
                  ]
                }
              example: |-
                {
                  "product": {
                    "id": "1",
                    "name": "Product One",
                    "description": "This is the full description of the product.",
                    "url": "http://example.com",
                    "image": "http://example.com/image.jpg",
                    "thumbnailUrl": "http://example.com/image-thumb.jpg",
                    "keywords": "western, cowboy",
                    "brand": "Brand Name",
                    "color": "Black",
                    "itemCondition": "New",
                    "manufacturer": "Manufacturer Name",
                    "model": "Black",
                    "sku": "SKU #",
                    "weight": "12 pounds",
                    "width": "12 inches",
                    "height": "12 inches",
                    "depth": "12 inches"
                  }
                }
      queryParameters:
        productId:
          type: integer
          description: The ID of the product
          required: true
          displayName: productId
    delete:
      description: Delete a single product
      responses:
        '200':
          body:
            application/json:
              schema: '{}'
              example: ''
      queryParameters:
        productId:
          type: integer
          description: The ID of the product
          required: true
          displayName: productId

```

```
[xmldom error]	invalid doc source 
@#[line:0,col:undefined]
[xmldom error]	invalid doc source 
@#[line:0,col:undefined]
[xmldom error]	invalid doc source 
@#[line:0,col:undefined]
```

**Comment by ddenisenko** [comment]
Duplicate of https://github.com/raml-org/raml-js-parser-2/issues/628

**Comment by dreamflyer** [comment]
will be published in v1.1.15.
